stonesoup/sensormanager/base.py

```diff
@@ -34,6 +41,33 @@ class SensorManager(Base, ABC):
                           "vector in the case of multi-objective optimisation. Metrics may be of "
                           "any type and in any units.")
 
+    take_sensors_from_platforms: bool = Property(
+        default=True, doc="Whether to include sensors that are on the "
+                          "platform(s) but not explicitly passed to the sensor manager. "
+                          "Any sensors not added "
+                          "will not be considered by the sensor manager or "
+                          "reward function.")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if self.platforms is None:
+            self.platforms = set()
+        if self._property_sensors is None:
+            self._property_sensors = set()
+
+    @sensors.getter
+    def sensors(self):
+        sensors = self._property_sensors
```

Comment:
Just thinking now, you probably shouldn't modify the original set, in case you remove a platform from the sensor manager for example.
```suggestion
        sensors = self._property_sensors.copy()
```